style/app.less

```diff
@@ -290,17 +290,6 @@ i.icon-spin {
 }
 
 
-
-.game-modal {
```

Comment:
Please remove unrelevant file. That issue should be a newcomer issue, while it is not labeled by gitmate-bot.